### DISCUSSION TITLE
Assets emit to same file name

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#12981] [Q&A]
My configuration for the chunk-maps is well-defined in the configuration as :

**`output: {
    path: path.resolve(config.build.client,"lib"),
    filename: "[name].js",
    chunkFilename: "[id].[chunkhash].js",
    library: "[name]"
  },`**

But yet, for some reason I still receive the following error which doesn't make any sense to me.

**`ERROR in Conflict: Multiple assets emit different content to the same filename client.map
`**

**Comment by sokra** [comment]
The error is not related to this piece of code.

**Reply by knightswatch3** [reply]
So what's the right place to look at ?

**Reply by alexander-akait** [reply]
plugins, maybe some plugins plugins duplicate assets

**Reply by knightswatch3** [reply]
So how do I nail this down ? Could you please point me the directions to start my debugging process ?

**Reply by alexander-akait** [reply]
You can try to comment plugins usage step by step and search the problem

**Reply by Jakobud** [reply]
Post your full webpack config

**Reply by knightswatch3** [reply]
@Jakobud Here is my webpack config:

const dllConfig = require("../dll.config.js");
const path = require("path");
const HTMLWebpackPlugin = require("html-webpack-plugin");
const config = require(path.resolve(__dirname, "..", "config.js"));
const webpack = require("webpack");
const SpritesmithPlugin = require("webpack-spritesmith");
const LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
const CircularDependencyPlugin = require("circular-dependency-plugin");
const packageJson = require("../../package.json");
const { query } = require("express");
console.log("====>", ...dllConfig.references());
module.exports = {
  output: {
    crossOriginLoading: "anonymous",
    path: config.build.client,
    filename: "[name][chunkhash].js", 
    sourceMapFilename: "[name].map",
    chunkFilename: `[id].chunk.${packageJson.version}.js`,
    publicPath: process.env.PUBLIC_PATH || "/"
  },
  stats: {
    colors: true,
    timings: true
  },
  resolve: {
    extensions: [".js", ".jsx", ".json"],
    modules: [path.resolve(__dirname, "..", "..", "src/client/client_modules"), "node_modules"],
    preferAbsolute: false
  },
  module: {
    rules: [
      {
        test: /\.jsx?$/,
        use: [
          {
            loader: "babel-loader",
            options:{
              plugins: ['syntax-dynamic-import']
            }
          },
          {
            loader: "import-glob"
          }
        ],
        // exclude: [path.resolve(__dirname, "../../node_modules")],
        include: [
          path.resolve(__dirname,"../../src/client"),
          path.resolve(__dirname,"../../config"),
          path.resolve(__dirname,"../../package.json")
        ]
      },
      {
        test: /\.txt$/,
        use: [
          {
            loader: "raw-loader"
          }
        ]
      },
      {
        test: /\.(png|jpg|jpeg|gif|svg)$/,
        use: [
          {
            loader: "file-loader",
            options: {
              // limit: 8192,
              name: "images/[name].[ext]"
            }
          }
        ]
      },
      {
        test: /\.(md)$/,
        use: [
          {
            loader: "raw-loader"
          }
        ]
      },
      {
        test: /\.(woff2|woff|eot|ttf|svg)$/,
        use: [
          {
            loader: "file-loader",
            options: {
              name: "fonts/[name].[ext]"
            }
          }
        ],
        include: /src\/client\/assets\/font/
      },
      {
        test: /\.(wav|mp3)$/,
        use: [
          {
            loader: "file-loader",
            options: {
              name: "sounds/[name].[ext]"
            }
          }
        ]
      }
    ]
  },
  recordsPath: path.resolve(config.build.client, "webpack.client.records"),
  plugins: [
    ...dllConfig.references(),
    new LodashModuleReplacementPlugin({
      shorthands: false,
      cloning: false,
      currying: false,
      caching: true,
      collections: true,
      exotics: false,
      guards: false,
      metadata: false,
      deburring: false,
      unicode: false,
      chaining: true,
      memoizing: false,
      coercions: true,
      flattening: true,
      paths: true,
      placeholders: false
    }),
    new CircularDependencyPlugin({
      exclude: /saga|node_modules/,
      failOnError: true
    }),
    // new webpack.optimize.ModuleConcatenationPlugin(),
    new webpack.HotModuleReplacementPlugin(), 
    new SpritesmithPlugin({
      src: {
        cwd: config.client.sprite.src,
        glob: "*.png"
      },
      target: {
        image: config.client.sprite.dest.image,
        css: [
          config.client.sprite.dest.css,
          [
            config.client.sprite.dest.json,
            {
              format: "json_texture"
            }
          ]
        ]
      },
      apiOptions: {
        cssImageRef: "~sprite.png"
      }
    }),
    new HTMLWebpackPlugin({
      title: config.name,
      template: config.client.html_index_template_path,
      favicon: config.client.favicon,
      inject: "body",
      extraScripts: dllConfig.entryNames()
    })
  ]
};
